test/runnable/testscope2.d

```diff
@@ -29,7 +29,7 @@ pragma(msg, "foo4 ", typeof(&SS.foo4));
 
 void test3()
 {
-    version (all)
+    version (none)
```

Comment:
You could change the following checks to `static assert`s to make these `writeln`s obsolete.